source/_integrations/persistent_notification.markdown

```diff
@@ -38,6 +36,17 @@ action:
     title: "Custom subject"
 ```
 
+or if you want to show some runtime information
+
+```yaml
+action:
+  service: persistent_notification.create
+  data_template:
+    title: >
+      Thermostat is {{ state_attr('climate.thermostat', 'hvac_action') }}
+    message: "Temperature {{ state_attr('climate.thermostat', 'current_temperature') }}"
+```
```

Comment:
````suggestion
{% raw %}

```yaml
action:
  service: persistent_notification.create
  data_template:
    title: >
      Thermostat is {{ state_attr('climate.thermostat', 'hvac_action') }}
    message: "Temperature {{ state_attr('climate.thermostat', 'current_temperature') }}"
```

{% endraw %}
````

Reply:
Strange, I think I checked and there was no `{% raw %}` around the existing example so I just copied it.  How could that happen, do you know?